Userland/Applications/PixelPaint/ImageEditor.cpp

```diff
@@ -351,6 +352,11 @@ void ImageEditor::context_menu_event(GUI::ContextMenuEvent& event)
 
 void ImageEditor::keydown_event(GUI::KeyEvent& event)
 {
+    if (event.key() == Key_Alt) {
+        set_old_tool(m_active_tool);
+        set_active_tool(new PickerTool());
```

Comment:
What if the currently active tool is already the picker tool?

Reply:
Alt key then makes no difference. The pointer changes to the "hotkey instance" of PickerTool but no visible changes to the user.